### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Release 3.0.0
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | @PrestaShopCorp
| How to test?      | Release Test

### CODE REVIEW

**Review by Hlavtox — APPROVED** [review]

**Comment by kpodemski** [comment]
@Progi1984 

I'm not sure about the versioning. If we change min compact to the next major, maybe we should introduce v3? ;-) What do you think?

**Review by kpodemski — DISMISSED** [review]
I'll block it for the moment. I strongly believe we should bump to v3 👍🏻 

I'll get back to you as soon as possible.

**Comment by Progi1984** [comment]
@kpodemski If a decision is took, please create an ADR for the future.

File: config.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <module>
     <name>blockwishlist</name>
     <displayName><![CDATA[Wishlist block]]></displayName>
-    <version><![CDATA[2.1.2]]></version>
+    <version><![CDATA[2.2.0]]></version>
```
Comment (kpodemski) [inline config.xml:5]:
```suggestion
    <version><![CDATA[3.0.0]]></version>
```

File: blockwishlist.php [hunk 2]
```diff
@@ -72,7 +72,7 @@ public function __construct()
     {
         $this->name = 'blockwishlist';
         $this->tab = 'front_office_features';
-        $this->version = '2.1.2';
+        $this->version = '2.2.0';
```
Comment (kpodemski) [inline blockwishlist.php:75]:
```suggestion
        $this->version = '3.0.0';
```

**Review by Hlavtox — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]
thanks @Progi1984 ❤️

**Comment by kpodemski** [comment]
Waiting for https://github.com/PrestaShop/blockwishlist/pull/211

**Review by florine2623 — COMMENTED** [review]
Hello @Progi1984 ,

I can't upgrade properly on `1.7.7.8` and on `8.0.0`. :x:
Screenshot 2023-02-23 at 09 54 32


On branch `8.1.x` and `develop`, the upgrade went well. ✅ 
Screenshot 2023-02-23 at 09 52 35

On PS `8.0.0` and with branch `8.1.x` and `develop`, I'm using Distribution API Client v1.0.2

Could you check the upgrade please ?
Thanks!

**Review by florine2623 — APPROVED** [review]
As discussed with @Progi1984 ,
The module blockwishlist is only compatible with v8 : https://github.com/PrestaShop/blockwishlist/pull/207

Tested on PS 8.0.1, 8.1.x and develop.

https://github.com/PrestaShop/blockwishlist/pull/193 ✅ 
https://github.com/PrestaShop/blockwishlist/pull/189 ✅ 
https://github.com/PrestaShop/blockwishlist/pull/186 ✅ 
https://github.com/PrestaShop/blockwishlist/pull/205 ✅ 
https://github.com/PrestaShop/blockwishlist/pull/207 ✅ 

It is QA ✅ 
Thanks !!

**Comment by Hlavtox** [comment]
🚀

**Pull request merged**